opencog/nlp/chatbot/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
-ADD_GUILE_MODULE (chatbot.scm
+ADD_GUILE_MODULE (FILES
+	chatbot.scm
```

Comment:
You are right. Since `chatbot.scm`  has file name equal to the basename of `MODULE_DESTINATION`, ADD_GUILE_MODULE automagically configures the install directory to be one directory lower. It doesn't prevent developers from using the function similar to the `INSTALL` command.